packages/amazonq/src/app/inline/sessionManager.ts

```diff
@@ -29,13 +31,15 @@ export class SessionManager {
         startPosition: vscode.Position,
         firstCompletionDisplayLatency?: number
     ) {
+        const diagnosticsBeforeAccept = getDiagnosticsOfCurrentFile()
         this.activeSession = {
             sessionId,
```

Comment:
you need to wait until the language server can properly ingest this field before you merge it